app/templates/components/request/config.hbs

```diff
@@ -0,0 +1,34 @@
+<div class="request-config">
+  <div class="header" {{on 'click' (action 'toggle')}}>
+    <SvgImage @name="job-name-icon" @class="icon" @width=14 @height=14 />
+    <span class="vertical-align">
+      Build Config
+    </span>
+    <div class="tools">
+      {{#if this.expanded}}
+        <Request::CopyButton
+          @copyText={{this.formattedConfig}}
+          @success={{action "copied"}}
+        >
+          <SvgImage @name="icon-copy" @class="icon" />
+          <span>
+            {{this.buttonText}}
+          </span>
+        </Request::CopyButton>
+      {{/if}}
+      <SvgImage
+        @name="icon-dropdown-arrow"
+        @class="icon-toggle icon-{{this.status}}"
+      />
+    </div>
+  </div>
+  {{#if this.expanded}}
+    <div class="position-relative">
+      <CodeBlock
+        @language="yaml"
+        class="codedisplay line-numbers"
+        data-test-config="true"
+      >{{this.formattedConfig}}</CodeBlock>
```

Comment:
I'd suggest to split it to multiple lines:
```suggestion
      >
        {{~this.formattedConfig~}}
      </CodeBlock>
```
The `~` should take care of whitespaces

Reply:
i think that's what i had initially, but this would add extra whitespace around the formatted config, which would then show up in the code block (i.e. a `pre` tag), so i believe we need to keep it as is, unfortunately.

Reply:
@svenfuchs are you sure you tried it with the `~`s around? They are crucial, as this is what tells Handlebars to remove whitespaces around the expression.

Reply:
aaah, no, i haven't. i've missed those in your suggestion, too. 

done: https://github.com/travis-ci/travis-web/pull/2399/commits/56232b70163954d0ad566b23789dd2e7bd2b835e